bcda/models/models.go

```diff
@@ -168,6 +169,30 @@ func (job *Job) GetEnqueJobs(resourceTypes []string, since string, newBeneficiar
 	return enqueJobs, nil
 }
 
+func setJobPriority(acoID string, resourceType string, sinceParam bool) int {
+	var priority int
```

Comment:
Maybe a comment in here to indicate that low number equals high priority.

Reply:
done